servatrice/src/servatrice_database_interface.cpp

```diff
@@ -120,7 +120,27 @@ bool Servatrice_DatabaseInterface::execSqlQuery(QSqlQuery *query)
 
 bool Servatrice_DatabaseInterface::usernameIsValid(const QString &user)
 {
-    static QRegExp re = QRegExp("[a-zA-Z0-9_\\.-]+");
+    int maxNameLength = settingsCache->value("users/maxnamelength").toInt();
+    int minNameLength = settingsCache->value("users/minnamelength").toInt();
```

Comment:
Can you add default values equal to defaults defined in the ini?
So that if someone upgrades servatrice but forgets to update his configuration, it will get same values instead of 0, 0.
The same applies to the bool variables following.